blocklist-client/src/config.rs

```diff
@@ -1,19 +1,35 @@
+use config::{Config, ConfigError, Environment, File};
+use once_cell::sync::Lazy;
 use serde::Deserialize;
 
-#[derive(Deserialize, Clone)]
+#[derive(Deserialize, Clone, Debug)]
 pub struct Settings {
     pub server: ServerConfig,
     pub risk_analysis: RiskAnalysisConfig,
 }
 
-#[derive(Deserialize, Clone)]
+#[derive(Deserialize, Clone, Debug)]
 pub struct ServerConfig {
     pub host: String,
     pub port: u16,
 }
 
-#[derive(Deserialize, Clone)]
+#[derive(Deserialize, Clone, Debug)]
 pub struct RiskAnalysisConfig {
     pub api_url: String,
     pub api_key: String,
 }
+
+pub static SETTINGS: Lazy<Settings> =
+    Lazy::new(|| Settings::new().expect("Failed to load configuration"));
+
+impl Settings {
+    pub fn new() -> Result<Self, ConfigError> {
+        let mut cfg = Config::new();
+        cfg.merge(File::with_name("./src/config/default"))?;
+        let env = Environment::with_prefix("APP").separator("__");
```

Comment:
Why the double underscore? Also, should we consider using a more specific prefix than "APP"? 
```suggestion
        let env = Environment::with_prefix("BLOCKLIST_CLIENT").separator("_");
```

Reply:
Sure, I'll update the prefix.

As for the separator, the nested configs wouldn't get picked up with a single underscore - [Related thread](https://github.com/mehcode/config-rs/issues/104#issuecomment-490971742)
I'll poke around with it a bit more but I run it like this:
` APP_server__host=127.0.0.1 APP_server__port=8080 APP_risk_analysis__api_url=https://chainalysis-api.com/ APP_risk_analysis__api_key=your_api_key cargo run`

Reply:
Okay, interesting. Are the env variables case sensitive? I'd expect it to be `APP_SERVER__HOST` and not `APP_server__host`.

Reply:
Specified it correctly in the README.md now.